resources/assets/css/_components.css

```diff
@@ -1,10 +1,10 @@
 /** Dropdowns **/
 .dropdown-entry {
-    @apply flex w-full items-center font-semibold text-left px-8 py-4 leading-5 transition duration-150 ease-in-out;
+    @apply flex w-full items-center font-medium text-left px-8 py-4 leading-5 transition duration-150 ease-in-out;
```

Comment:
we don't really use `font-medium`, there's not even a font associated with it in the css so this won't do anything. Oleg should have refrained from using medium in designs as it was only used once before and would mean we need to load an additional font for just one thing which is overkill :olegcornercry: